clouds/pom.xml

```diff
@@ -78,6 +78,7 @@
                         <excludes>
                             <!-- These should not be run alone since they are supposed to be run in a suite -->
                             <exclude>**/suite/tests/*.java</exclude>
+                            <exclude>**/suitetests/*.java</exclude>
```

Comment:
is this expected? I do not see such folder.

Reply:
It is in `azure.armTemplates.di.suitetests`. It is there to exclude those test classes so that they are note executed alone but only suite including them is executed.

Reply:
Ok